### PR TITLE
Clarified docs about week numbering

### PR DESCRIPTION
**Pull request by alasdairnicol**

### CODE REVIEW

**Comment by alasdairnicol** [comment]
I'm not sure that this is the best possible wording, but I wanted to change "the first week starts on or before Thursday." -- we've already said that weeks start on Monday.

File: docs/ref/models/querysets.txt [hunk 1]
```diff
@@ -2888,7 +2888,7 @@ in the database <database-time-zone-definitions>`.
 
 For date and datetime fields, return the week number (1-52 or 53) according
 to `ISO-8601 <https://en.wikipedia.org/wiki/ISO-8601>`_, i.e., weeks start
-on a Monday and the first week starts on or before Thursday.
+on a Monday and the first week contains that year's first Thursday.
```
Comment (timgraham) [inline docs/ref/models/querysets.txt]: that -> the

File: docs/ref/models/database-functions.txt [hunk 2]
```diff
@@ -222,8 +222,8 @@ returned when this timezone is active will be the same as above except for:
     The ``week`` ``lookup_type`` is calculated based on `ISO-8601
     <https://en.wikipedia.org/wiki/ISO-8601>`_, i.e.,
     a week starts on a Monday. The first week is the one with the majority
-    of the days, i.e., a week that starts on or before Thursday. The value
-    returned is in the range 1 to 52 or 53.
+    of the days, i.e., the week that contains that year's first Thursday.
```
Comment (timgraham) [inline docs/ref/models/database-functions.txt]: I think the part about "a majority of days" could be removed: "The first week of a year is the one that contains the year's first Thursday."

Comment (timgraham) [reply docs/ref/models/database-functions.txt]: Okay, I took your suggestion and used language from the wikipedia article.

Comment (alasdairnicol) [reply docs/ref/models/database-functions.txt]: Thanks @timgraham and @akulakov for the improvements. Looks good to me 👍

**Pull request merged**